driver/razerkbd_driver.c

```diff
@@ -274,6 +274,7 @@ static void razer_get_report_params(struct usb_device *usb_dev, uint *report_ind
     case USB_DEVICE_ID_RAZER_DEATHSTALKER_V2_PRO_WIRED:
     case USB_DEVICE_ID_RAZER_BLACKWIDOW_V4:
     case USB_DEVICE_ID_RAZER_BLACKWIDOW_V4_PRO:
+    case USB_DEVICE_ID_RAZER_BLACKWIDOW_V4_75PERCENT:
```

Comment:
I wonder if renaming the identifier to [...]_75PCT is better than the current [...]_75PERCENT. What do you think?

Reply:
Yes, I agree using the unit would be much better than a full word.